src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -614,8 +614,7 @@ const CreateImageWizard = () => {
 
   // Only allow oscap to be used in Beta even if the flag says the feature is
   // activated.
-  const oscapFeatureFlag =
```

Comment:
Is there anything else I might have missed for us to move this to `prod-stable`?